src/components/ChallengeModal/ChallengeModal.tsx

```diff
@@ -595,12 +601,26 @@ export class ChallengeModal extends Modal<Events, ChallengeModalProperties, any>
             ["challenge.game.private", ev],
             ["challenge.game.ranked", false],
         ]);
-    update_rengo = (ev) =>
+    update_rengo = (ev) => {
         this.upstate([
             ["challenge.game.rengo", ev],
             ["challenge.game.ranked", false],
             ["challenge.game.handicap", 0],
         ]);
+    };
+    update_rengo_casual = (ev) => {
+        this.upstate("challenge.game.rengo_casual_mode", ev);
+        // brute force ensure that they are using simple time in casual mode
+
+        this.setState({
```

Comment:
We should probably default the speed to whatever the current speed is, so if they have correspodence set, we set simple correspondence with whatever time, etc. Bonus points for remembering their last rengo time settings :)

Reply:
Yeah, indeed we should.  Do we have to?  The full solution may include fixing the current bug with saving the different speed settings, which is daunting...

Reply:
... note that it's only a something you experience the first time, or if you swap between casual and strict rengo Otherwise it just remembers that you like casual rengo at the speed you set last time.

Reply:
Yay, with the other fix, this is no longer a thing